src/templates/components/head/index.js

```diff
@@ -1,10 +1,19 @@
 import { h } from 'preact';
 
-const head = ({ title, meta, children }) => <head>
-    <meta charSet="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no" />
-    <link rel="shortcut icon" href={`/static/img/favicon.ico`} />
-    <title>Storm UI Patterns - {title}</title>
+const head = ({
```

Comment:
Really like the inclusion of all <head> elements as parameters. Much clear / more explicit.